packages/events/src/service/indexing/indexing.ts

```diff
@@ -146,7 +146,7 @@ function mapFieldTypeToElasticsearch(
         type: 'object',
         properties: {
           age: { type: 'double' },
-          asOfDate: { type: 'date' }
+          asOfDateRef: { type: 'keyword' }
```

Comment:
We probably need to store the `asOfDate` or `dob` in elastic index. The queries won't work otherwise right?